app/webpacker/controllers/guest_checkout_controller.js

```diff
@@ -34,4 +38,21 @@ export default class extends Controller {
   usingGuestCheckout() {
     return sessionStorage.getItem(this.sessionValue) === this.distributorValue;
   }
+
+  handlePageUnload(event) {
+    event.preventDefault()
+
+    event.returnValue = 'I18n.t("admin.unsaved_confirm_leave")';
+    return 
```

Comment:
This statement returns `undefined` and it's also the default behavior if we don't return anything from the JS function.
```suggestion
```